integration-tests-ex/cases/cluster/Query/docker-compose.yaml

```diff
@@ -0,0 +1,98 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
The `cluster.sh` file says we're using `BatchIndex` for this test, yet we have a custom `docker-compose.yaml` file that isn't being used.

Reply:
Thanks for pointing this out. Had added them while doing some experimenting and forgot to remove it while pushing the latest change.